### ISSUE TITLE
Invariant Violation: View config getter callback for component `RemoteVideoView` must be a function (received `undefined`)

### ISSUE THREAD

**Issue by jie-meng** [microsoft/omnichannel-voice-video-calling-react-native#40]
I use React Native 0.68.1 on my Android device.
Follow README.md make
```
<View style={display ? styles.videoContainer : styles.noDisplay}>
  <RemoteVideoView />
</View>
```
But got this error:
Invariant Violation: View config getter callback for component `RemoteVideoView` must be a function (received `undefined`)

**Comment by elopezanaya** [comment]
Are you trying to run the sample app?

**Comment by jie-meng** [comment]
No, I create a totally new React Native project. Follow README.md steps.
omnichannel-chat-sdk works.
voice-video-calling-react-native-sdk fails.

Here's my project.
https://github.com/jie-meng/TestRN

The code is in App.js.

Could you help me to check what's wrong with the code? Thanks!
